helm/prometheus-agent/charts/prometheus-agent/files/watchdog.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+### Config
+
+metrics_url="localhost:9090/metrics"
+overload_tolerance="${WATCHDOG_OVERLOAD_TOLERANCE:-2}" # allow up to x times max shards
+
+
+exit_error() {
+    echo "ERROR: $*"
+    exit 1
+}
+
+### Default liveness probe:
+#    livenessProbe:
+#      failureThreshold: 6
+#      httpGet:
+#        path: /-/healthy
+#        port: http-web
+#        scheme: HTTP
+#      periodSeconds: 5
+#      successThreshold: 1
+#      timeoutSeconds: 3
+default_liveness() {
+  wget --timeout 3 --tries 3 -q -O /dev/null http://localhost:9090/-/ready
```

Comment:
hmm, nice catch.

Short answer: I don't know.

Long answer:
I took note of the default (httpGet) liveness probe before changing anything.
Then I switched to "ListenLocal" mode, which used a "command" probe.
And I copied the command from the "command" probe to put it in my script.

But now I looked at the code:
* [definition of the probes](https://github.com/prometheus-operator/prometheus-operator/blob/ef1958adf24c8c793e42c7a5f75f464a7b8c50d8/pkg/prometheus/statefulset.go#L616)
* [comment about which path to use](https://github.com/prometheus-operator/prometheus-operator/blob/ef1958adf24c8c793e42c7a5f75f464a7b8c50d8/pkg/prometheus/statefulset.go#L649)

...it looks clear I should stick to `/-/healthy`.
I'll fix it.

Reply:
...about the number of tries: it's also kept in the probe definition.
I added a retry to `wget` because, as we run it 3 times in the script, chances that it fails are higher.

Reply:
Removed the misleading comments.